_health-care/_js/components/form-elements/ErrorableRadioButtons.jsx

```diff
@@ -57,15 +57,15 @@ class ErrorableRadioButtons extends React.Component {
       }
       const checked = storedValue !== undefined && optionValue === storedValue ? 'checked=true' : '';
       return (
-        <div key={reactKey++} className="radio-buttons-inline">
+        <div key={reactKey++} className="hca-radio-buttons">
           <input
               checked={checked}
-              id={optionValue}
-              name={this.inputId}
+              id={`${this.inputId}-${index}`}
+              name={`${this.inputId}-${index}`}
```

Comment:
Why is the `name` getting a unique `index`? Shouldn't both inputs share the same `name`?

For example:

``````
<ul class="usa-unstyled-list">
      <li>
        <input id="pea-soup" type="radio" checked="" name="soup" value="pea">
        <label for="pea-soup">Pea Soup</label>
      </li>
      <li>
        <input id="chicken-noodle" type="radio" name="soup" value="chicken-noodle">
        <label for="chicken-noodle">Chicken Noodle Soup</label>
      </li>
      <li>
        <input id="tomato" type="radio" name="soup" value="tomato">
        <label for="tomato">Tomato Soup</label>
      </li>
    </ul>
```